x-pack/plugins/fleet/server/collectors/register.ts

```diff
@@ -49,10 +49,11 @@ export function registerFleetUsageCollector(
     schema: {
       agents_enabled: { type: 'boolean' },
       agents: {
-        total: { type: 'long' },
-        online: { type: 'long' },
-        error: { type: 'long' },
+        total_enrolled: { type: 'long' },
+        healthy: { type: 'long' },
+        unhealthy: { type: 'long' },
```

Comment:
Can we add descriptions to the schema fields? using `_meta.description`. This would serve as documentation to the fields and helps people who use this data to understand these fields.